b0xing/app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,4 @@
 <resources>
-    <string name="app_name">My Application</string>
-
-    <string name="hello_world">Hello world!</string>
+    <string name="app_name">boxr</string>
```

Comment:
That's OK. But are you sure that's the most important part of this entire pull request?